Makefile

```diff
@@ -49,6 +49,16 @@ ruby: docker-image
 		--entrypoint bash ${PROTOC_IMAGE} \
 		-c "cd ./gen/pb-ruby && protoc -I/opt/include -I../../protos/ --ruby_out=lib ../../protos/*.proto"
 
+rust: docker-image
+	@echo "Generating rust protobuf files"
+	docker run \
+		--platform linux/amd64 \
+		-v ${PWD}:/defs \
+		-e "RUST_BACKTRACE=1" \
+		--entrypoint bash ${PROTOC_IMAGE} \
+		-c "cd gen/pb-rust && cargo build"
```

Comment:
This isn't the best. Because some of the includes are in `/opt/`, any cargo commands have to be threaded through this (or parallel) Makefile rules.